.github/workflows/reusable-create-pypi-release.yml

```diff
@@ -0,0 +1,77 @@
+name: Reusable workflow to create and publish pypi release
+
+
+on:
+  workflow_call:
+    inputs:
+      release_url:
+        required: true
+        type: string
+      os:
+        required: true
+        type: string
+      py-version:
+        required: true
+        type: string
+      package_version:
+        required: true
+        type: string
+    secrets:
+      PLATOMO_BUILDER_ACCESS:
+        required: true
+
+jobs:
+  build-n-publish:
+    name: Build and upload release to TestPyPI
+    runs-on: ${{ input.os }}
+    strategy:
+      fail-fast: True
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.11"]
+    environment:
+      name: testpypi
+      url: inputs.release_url
+    permissions:
+      actions: read
+      contents: read
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ inputs.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ inputs.python-version }}
+      - name: Check-out platomo-build repository
+        uses: actions/checkout@v4
+        with:
+          repository: platomo/platomo-build
+          token: ${{ secrets.PLATOMO_BUILDER_ACCESS }}
+          path: platomo-build
+      - name: Display dist content
```

Comment:
Why do we need `Display dist content` and `Remove file`? This might be a relict of the prototype branch. In the prototype branch it was necessary for debugging and because the platomo-build run before and the dist folder contained data.